.vscode/settings.template.json

```diff
@@ -54,26 +55,15 @@
   "[makefile]": {
     "editor.insertSpaces": false
   },
-  "python.testing.pytestEnabled": true,
```

Comment:
Isn't there a way that we can make use of this? I think the vscode integration of pytest is awesome. I think the problem is that vs code is trying to discover all tests and this fails because not all dependencies are there for **all** tests. But if one could enable this for the current working directly or something like that, then it would be great

Reply:
OK, I was just playing a bit around with this and it seems it is possible to configure vscode to handle our pytests correctly. What needs to be added is the [option](https://code.visualstudio.com/docs/python/testing#_pytest-configuration-settings) `pytest.testing.pytestArgs`. E.g. if I set 
```
"python.testing.pytestArgs": ["services/api-server/tests/unit"]
```
then I can run all the api-server unit tests through the GUI. I think this is actually very useful. For instance it is very easy to run a single test with the debugger without marking the test with the `testit` marker.

Reply:
Let's check offline what is more convenenient.